kuttl-test.yaml

```diff
@@ -3,10 +3,8 @@ kind: TestSuite
 manifestDirs:
```

Comment:
I'm curious, why is there `kudo-test.yaml` as well as `kuttl-test.yaml` and both have the same content? Looks like one of these should be removed. Anyways, this isn't your work, could also be done later.

Reply:
1. `kudo-test.yaml.tmpl` is templated *now* becasue KUDO CI `sed`s the right kudo image `%version%`. For the same reason it has the additional `--kudo-image kudobuilder/controller:%version% --kudo-image-pull-policy Never` options. 
2. The `kuttl-test.yaml` is for operator internal tests and doesn't need to jump through all the hoops because it `pull`s a tagged KUDO image.

Reply:
I see. Still, there's now two places to add new tests, it would be better to have a single one. Well, even better would be to collect test cases automatically from the repo :). But having a single place to define tests is a start. Can't we use `kudo-test.yaml.tmpl` in both cases and have to operator internal test set `%version%` to a tagged one?

Reply:
Note, that current *master* already has two files. But sure, we could consolidate them but that's a followup PR 😉

Reply:
Also, note that `kudo-test.yaml` might have different options as we keep it in sync with KUDO master. E.g. this PR removes the `--webhook InstanceValidation` which we're about to remove in kudo/#1497 but it will still be present in `kuttl-test.yaml` as long as it pulls the `v0.13.0` image. That a point for the two-files solutions 🤷‍♀️

Reply:
Yeah, good point.